README.md

```diff
@@ -10,14 +10,123 @@
+
+## What Ruby on Rails is not
+* **A programming language** - This is one of the most common misconceptions, Ruby on Rails is not a programming language, instead it is a set of code libraries built in Ruby.
+* **A slow framework** - Due to the fact that Rails is one of the most straightforward frameworks to learn, it can lead to a number of poor coding practices from beginners. However if built properly, Rails projects can be as fast as any other framework. Furthermore, with Rails service based architecture it makes it a perfect candidate for microservice applications, which can be some of the fastest and best performing applications on the web.
+
+# Creating your first Rails project
+
+*Before you continue, this assumes that you have Ruby, RubyGems, and Bundler installed on your system*
+
+## Installing the Rails Gem
+
+As mentioned above, Rails is simply a Ruby Gem, to install it on your system, run the following command in the terminal of your computer:
+
+```gem install rails```
+
+Depending on your system configuration you may need to prepend the command with ```sudo``` to install the gem as the root user. Once the gem is installed you can create Rails applications!
+
+## Generating a new Rails application
+
+Our application is going to be called BlogFlash, to create the application, run the following command:
+
+```rails new blog-flash```
+
+There are a number of common naming conventions for Rails app names, typically you will want to use all lower case letters, separated by '-', as shown in our ```blog-flash``` naming structure. In the same way that there are rules for naming methods, variables, classes, etc. in Ruby, there are naming rules for application names. Since the application name is used as the app constant and throughout the application the best approach is to keep your naming simple and to follow a standard naming practice.
+
+## Rails File Structure Overview
+
+Since you will be working with this file structure on a daily basis, it is very important to understand and become familiar with the file system. Below is a breakdown for each directory:
+
+* app – contains the models, views, and controllers, along with the the rest of the core functionality of the application. This is the one directory where you can make a change and not have to restart the Rails server. The majority of your time will be spent working in this directory. In addition to the full MVC structure, this directory also contains the full Rails asset pipeline directories, such as: css files, javascripts, images, fonts, etc.
```

Comment:
They don't know the term "rails asset pipeline". Just mention that this directory also contains all of the other non ruby files that will get served to a user such as css, javascript, images, fonts, etc.